alleycats-core/src/main/scala/alleycats/std/set.scala

```diff
@@ -30,8 +30,12 @@ trait SetInstances {
   // contain three. Since `g` is not a function (speaking strictly)
   // this would not be considered a law violation, but it still makes
   // people uncomfortable.
-  implicit val alleyCatsStdSetMonad: Monad[Set] =
-    new Monad[Set] {
+  //
+  // If we accept Monad for Set, we can also have Alternative, as
+  // Alternative only requires MonoidK (already accepted by cats-core) and
+  // the Applicative that comes from Monad.
+  implicit val alleycatsStdInstancesForSet: Monad[Set] with Alternative[Set] =
```

Comment:
This name follows [the Cats implicit-naming guidelines](https://typelevel.org/cats/guidelines.html#-implicit-naming), I think.